rules-tests/TypeDeclaration/Rector/StmtsAwareInterface/DeclareStrictTypesRector/Fixture/some_class.php.inc

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\StmtsAwareInterface\DeclareStrictTypesRector\Fixture;
+
+function someFunction()
+{
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\StmtsAwareInterface\DeclareStrictTypesRector\Fixture;
+
+declare(strict_types=1);
```

Comment:
it should be above namespace, or it will cause fatal error

```
Fatal error: strict_types declaration must be the very first statement
```

see https://3v4l.org/UoVX9